backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/upsertPrices.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Create or update using prices API
+  Using default ad_orgId 1000000
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+
+  Scenario: Create price list version from external ref
+    Given metasfresh contains M_PricingSystem
+      | Identifier | Name                | Value                | OPT.Description            | OPT.IsActive |
```

Comment:
this is not an identifier but a `M_PricingSystem_ID`.
When i used the term identifier in my tests, added the records to a map, and used the identifier as my key.
It's fine for me to use this approach of *setting* the ID explicitly (i simply didn't think of it), but then please use the actual column name like `M_PriceList_ID`, `M_ProducT_ID` etc